prismic-model/lintPrismicData.ts

```diff
@@ -29,6 +33,13 @@ type ErrorProps = {
   errors: string[];
 };
 
+const { isGitHubAction } = yargs(process.argv.slice(2))
```

Comment:
See later comment about removing the need to worry about whether running in GHA or not.